rfcs/52-unstable_features.md

```diff
@@ -0,0 +1,82 @@
+- A small feature with a clear implementation? Go ahead and open that PR!
+- A large feature with an open design space that could use some discussion about how to implement it? A RFC would be good.
+- Something in between that would benefit from iterative work? Here come the unstable feature workflow for you!
+
+You will need to open a tracking issue to explain the new feature you want to add. If it looks good, you'll be ready to open that PR with a feature gate for your changes!
+
+## Implementation strategy
+
+The new workflow for this kind of feature would be:
+- Submit an issue explaining the feature you want to add under a feature gate. This will be the tracking issue for stabilization once approved. This issue gets a new label `S-Stabilization`.
+- If approved, submit a PR with the new feature and the feature gate
+- This PR can be merged by someone with the merge rights on the related part of Bevy as soon as it's `S-Ready-For-Final-Review`, without an approval by the main Bevy maintainers.
+- Subsequent PRs on the feature must be linked to the tracking issue and follow the same process.
+- Once the feature is finished, enter the stabilization period. Open a PR that will remove the feature gate. This PR needs to be approved by the main Bevy maintainers.
+
+The feature gate should follow the following convention for its name: `unstable-<#tracking-issue-number>-<feature-name>`.
+
+An existing PR can be retroffited in this process on the suggestion of Bevy community members, and if the PR author agrees.
+
+The RFCs and the unstable features are closely related. An RFC can be implemented as an unstable feature to help resolve issues that need more experimentation, or split a large implementation. An unstable feature could require an RFC to reach stabilization. For example:
+- RFC to Unstable Feature
+    - Open a RFC, reaching a design
+    - Implement part of the RFC as an unstable feature
+    - Feedback into the RFC with the gained knowledge of implementation and usage
+- Unstable Feature to RFC
+    - Open an tracking issue, start implementing it
+    - Part of the unstable feature is larger than expected with several deisgn possibilities
+    - Open a RFC, reaching a decision on the design
+    - Implement the RFC for the unstable feature
+
+
+
+Each PR will still need to be approved by Bevy community members, and would still need to have the same level of quality as other PRs merged.
+
+The PRs for the initial feature, various changes during its finalization and for the stabilization can be opened by different persons.
+
+The feature gate should not be exposed on the main `bevy` crate, but only on the subcrates where it is relevant. It can be enabled as a user by adding dependencies directly on the subcrates and enabling the feature. The tracking issue should list the subcrates impacted.
+
+For organization and discovery, a GitHub project can be used to track the stabilization issues.
+
+## Drawbacks
```

Comment:
I think another drawback could be that such unstable features may need constant upkeep based on their dependencies. And it might be difficult for the author(s) and other contributors to know that a stable change affects an unstable one.

Although this might just be the expectation.

Reply:
good point, added!